lib/block.ml

```diff
@@ -215,32 +211,6 @@ let set_variables t = t.set_variables
 
 let unset_variables t = t.unset_variables
 
-let explicit_required_packages t = t.required_packages
-
-let require_re =
-  let open Re in
-  seq [ str "#require \""; group (rep1 any); str "\"" ]
-
-let require_from_line line =
```

Comment:
Since I am a bit new to this project I'm a bit confused. I don't understand what removing the rule command has to do with removing support for `require`? Or am I misunderstanding this?

Reply:
This code was only used to set up the `deps` field of dune rules (as part of the rule command) and it doesn't impact the interpretation of `require` directive in the blocks